app/views/admin/loans/timeline/_batch_actions.html.erb

```diff
@@ -12,14 +12,14 @@
   </li>
 
   <li>
-    <a id="adjust-dates" class="action" data-toggle="modal"
-      data-target="#adjust-dates-modal">
+    <a class="action adjust-dates" data-toggle="modal"
+      data-target="#<%= locals[:modal_id] %>">
```

Comment:
I am not sure why this is happening here. `modal_id` does not work, but `locals[:modal_id]` does....